modules/users/server/models/user.server.model.js

```diff
@@ -18,7 +19,14 @@ var validateLocalStrategyProperty = function(property) {
  * A Validation function for local strategy password
  */
 var validateLocalStrategyPassword = function(password) {
-	return (this.provider !== 'local' || (password && password.length > 6));
+	return (this.provider !== 'local' || validator.isLength(password, 6));
```

Comment:
No, its just that I thought it was good for consistency to use validator where possible.